From 1a57fae51e0e016b2a57b466d48f61c75fbb6ebe Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Tue, 7 Apr 2015 14:31:34 +0800
Subject: [PATCH 0941/1594] MLK-10591-1 ARM: dts: add ipg clk node for imx6qdl

commit b041b5f575a521d38ee021970a1bd51444aeb59b from
git://git.freescale.com/imx/linux-2.6-imx.git

Add ipg clk node in gpc for kernel use.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/boot/dts/imx6qdl.dtsi |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl.dtsi b/arch/arm/boot/dts/imx6qdl.dtsi
index d83fef2..633fd0b 100644
--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -787,7 +787,8 @@
 					 <&clks IMX6QDL_CLK_GPU2D_CORE>,
 					 <&clks IMX6QDL_CLK_GPU2D_AXI>,
 					 <&clks IMX6QDL_CLK_OPENVG_AXI>,
-					 <&clks IMX6QDL_CLK_VPU_AXI>;
+					 <&clks IMX6QDL_CLK_VPU_AXI>,
+					 <&clks IMX6QDL_CLK_IPG>;
 				#power-domain-cells = <1>;
 			};
 
-- 
1.7.5.4

